mycore-base/src/main/java/org/mycore/frontend/servlets/MCRServlet.java

```diff
@@ -87,6 +91,16 @@
     private static final boolean ENABLE_BROWSER_CACHE = MCRConfiguration.instance().getBoolean(
         "MCR.Servlet.BrowserCache.enable", false);
 
+    private static final String SESSION_NETMASK_IPV4_STRING = MCRConfiguration.instance()
+        .getString("MCR.Servlet.Session.NetMask.IPv4", "255.255.255.255");
```

Comment:
these properties should also be part of mycore.properties and documentated there

Reply:
Done.